src/main/java/com/github/thedeathlycow/moregeodes/features/ModConfiguredFeatures.java

```diff
@@ -213,6 +213,35 @@ public class ModConfiguredFeatures {
                         0.3f
                 )
         ));
+
+        if (FabricLoaderImpl.INSTANCE.isModLoaded("ae2")) {
+            CERTUS_GEODE = register("certus_geode", new ConfiguredFeature<>(Feature.GEODE, new GeodeFeatureConfig(
+                    new GeodeLayerConfig(
+                            SimpleBlockStateProvider.of(Blocks.WATER.getDefaultState()),
```

Comment:
Should the filling be water and not air?

Reply:
Well, it does be said that meteors brought water on Earth, so following that idea, certus geodes would only be buried meteors